web/src/app.tsx

```diff
@@ -66,7 +43,50 @@ export default function App() {
             <Suspense fallback={<Loading />}>
               <NotificationBanner />
               <StyledRouter>
-                <AppRoutes />
+                <Routes>
```

Comment:
Separating entire `Routes` into another file might be good for maintainability and readability.

Reply:
This is true. I will do this as well in a separate PR!